### PR TITLE
Feature: Enabled tags by default and removed option to turn them off

### LINKED ISSUE #9586: Turn tags on by default and remove setting to turn it off
We're getting close to the point where tags can leave preview and we can remove the setting to disable it. We should hold off on this a bit longer but I'm opening this issue to keep track.

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved*
- Closes #9586 

**Details of Changes**
-Removed Tag ToggleSwitch
-Removed all the code that was executed at change of the toggle status
-Removed all the ifs that checked if tags were enabled (Now their code is always executed)

**Validation**
- Built and ran the app, everything worked fine

### CODE REVIEW

**Comment by yaichenbaum** [comment]
As noted in the issue, we're not ready to merge this just yet, let's keep this open in the meantime.

File: src/Files.Uwp/Strings/af/Resources.resw [hunk 1]
```diff
@@ -59,46 +59,46 @@
             : using a System.ComponentModel.TypeConverter
             : and then encoded with base64 encoding.
     -->
-  <xsd:schema xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata" id="root">
-    <xsd:import namespace="http://www.w3.org/XML/1998/namespace"/>
+  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
```
Comment (yaichenbaum) [inline src/Files.Uwp/Strings/af/Resources.resw:62]: Can you revert changes to the resource files? Removing the strings from the en-us resource file should be enough.

File: src/Files.Uwp/UserControls/SidebarControl.xaml [hunk 2]
```diff
@@ -36,7 +36,7 @@
             <ToggleMenuFlyoutItem IsChecked="{x:Bind ViewModel.ShowWslSection, Mode=TwoWay}" Text="{helpers:ResourceString Name=WSL}" />
             <ToggleMenuFlyoutItem
                 x:Name="SidebarContextMenuFileTagsSection"
-                x:Load="{x:Bind ViewModel.AreFileTagsEnabled, Mode=OneWay}"
+                x:Load="True"
```
Comment (yaichenbaum) [inline src/Files.Uwp/UserControls/SidebarControl.xaml:39]: x:Load defaults to `True` so this line can be removed.

File: src/Files.Uwp/Views/SettingsPages/Appearance.xaml [hunk 3]
```diff
@@ -177,7 +177,7 @@
                             </local:SettingsBlockControl.Icon>
                             <ToggleSwitch
                                 AutomationProperties.Name="{helpers:ResourceString Name=SettingsShowFileTagsSectionSwitch/AutomationProperties/Name}"
-                                IsEnabled="{Binding AreFileTagsEnabled}"
+                                IsEnabled="True"
```
Comment (yaichenbaum) [inline src/Files.Uwp/Views/SettingsPages/Appearance.xaml:180]: IsEnabled defaults to true so this line can be removed.

File: src/Files.Uwp/Views/SettingsPages/Experimental.xaml [hunk 4]
```diff
@@ -33,34 +33,7 @@
...
-
-            <local:SettingsBlockControl
-                Title="{helpers:ResourceString Name=SettingsEnableFileTags/Title}"
-                HorizontalAlignment="Stretch">
-                <local:SettingsBlockControl.Icon>
-                    <FontIcon Glyph="&#xE1CB;" />
-                </local:SettingsBlockControl.Icon>
-                <ToggleSwitch IsOn="{Binding AreFileTagsEnabled, Mode=TwoWay}" Style="{StaticResource RightAlignedToggleSwitchStyle}" />
-                <local:SettingsBlockControl.ExpandableContent>
-                    <local:SettingsBlockControl
```
Comment (yaichenbaum) [inline src/Files.Uwp/Views/SettingsPages/Experimental.xaml:45]: We still need the option to edit tags, can you add it back?

Comment (ferrariofilippo) [reply src/Files.Uwp/Views/SettingsPages/Experimental.xaml:45]: Ok no problem
Just one question: should I add this feature back in the settings or can I create a "Rename" command -that opens you the file- when you right click them in the sidebar?

Comment (yaichenbaum) [reply src/Files.Uwp/Views/SettingsPages/Experimental.xaml:45]: We want to add a proper UI that allows users to edit tags but in the meantime, we should just add the edit button back into settings. If you're interested in working on the edit UI, we should discuss it first and come up with a plan.

Comment (ferrariofilippo) [reply src/Files.Uwp/Views/SettingsPages/Experimental.xaml:45]: Ok

Comment (ferrariofilippo) [reply src/Files.Uwp/Views/SettingsPages/Experimental.xaml:45]: - I added the "Edit Tags" feature in the "Preference" section of "Settings"
- I removed faulty/not working options from the "Tags" on the Sidebar (Ex. "Open in another Window")
- I added this feature "allow the user to hide the "Tags" column from all folders at once" requested by @gave92 in a comment to #9586

Comment (yaichenbaum) [reply src/Files.Uwp/Views/SettingsPages/Experimental.xaml:45]: > I added the "Edit Tags" feature in the "Preference" section of "Settings"

This should still be under experimental for now 👍 

> I removed faulty/not working options from the "Tags" on the Sidebar (Ex. "Open in another Window")

We shouldn't remove them from the app, we should open an issue to track the bug.

Comment (yaichenbaum) [reply src/Files.Uwp/Views/SettingsPages/Experimental.xaml:45]: > I added this feature "allow the user to hide the "Tags" column from all folders at once" requested by @gave92 in a comment to

Thanks! Can you cherry pick this commit and open a separate PR for this?

Comment (yaichenbaum) [reply src/Files.Uwp/Views/SettingsPages/Experimental.xaml:45]: Thanks! Can you remove the commit from this branch?

Comment (ferrariofilippo) [reply src/Files.Uwp/Views/SettingsPages/Experimental.xaml:45]: If you mean the "Hide Tags Column" I've already removed it from last commit

File: src/Files.Uwp/Filesystem/ListedItem.cs [hunk 5]
```diff
@@ -133,8 +133,7 @@ public string[] FileTags
 
         public IList<FileTagViewModel> FileTagsUI
         {
-            get => UserSettingsService.PreferencesSettingsService.AreFileTagsEnabled ?
-                FileTagsSettingsService.GetTagsByIds(FileTags) : null;
+            get => FileTagsSettingsService.GetTagsByIds(FileTags);
```
Comment (yaichenbaum) [inline src/Files.Uwp/Filesystem/ListedItem.cs:136]: Should we still have the null check?

File: src/Files.Uwp/Helpers/ContextFlyoutItemHelper.cs [hunk 6]
```diff
@@ -291,7 +291,7 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseLayoutMenuItems(Curren
                             Text = "FileTags".GetLocalized(),
                             IsChecked = itemViewModel.IsSortedByFileTag,
                             Command = new RelayCommand(() => itemViewModel.IsSortedByFileTag = true),
-                            ShowItem = userSettingsService.PreferencesSettingsService.AreFileTagsEnabled,
+                            ShowItem = true,
```
Comment (yaichenbaum) [inline src/Files.Uwp/Helpers/ContextFlyoutItemHelper.cs:294]: Does ShowItem default to true? Just wondering if we can remove this line.

File: src/Files.Uwp/UserControls/InnerNavigationToolbar.xaml [hunk 7]
```diff
@@ -449,7 +449,7 @@
                             <ToggleMenuFlyoutItem
                                 IsChecked="{x:Bind ViewModel.IsSortedByFileTag, Mode=TwoWay}"
                                 Text="{helpers:ResourceString Name=FileTags}"
-                                Visibility="{x:Bind UserSettingsService.PreferencesSettingsService.AreFileTagsEnabled, Mode=OneWay}" />
+                                Visibility="Visible" />
```
Comment (yaichenbaum) [inline src/Files.Uwp/UserControls/InnerNavigationToolbar.xaml:452]: The default value is visible so this line can be removed.

File: src/Files.Uwp/UserControls/InnerNavigationToolbar.xaml [hunk 8]
```diff
@@ -473,7 +473,7 @@
                             <ToggleMenuFlyoutItem
                                 IsChecked="{x:Bind ViewModel.IsGroupedByFileTag, Mode=TwoWay}"
                                 Text="{helpers:ResourceString Name=FileTags}"
-                                Visibility="{x:Bind UserSettingsService.PreferencesSettingsService.AreFileTagsEnabled, Mode=OneWay}" />
+                                Visibility="Visible" />
```
Comment (yaichenbaum) [inline src/Files.Uwp/UserControls/InnerNavigationToolbar.xaml:476]: Same as above

File: src/Files.Uwp/UserControls/SidebarControl.xaml.cs [hunk 9]
```diff
@@ -261,21 +261,21 @@ private List<ContextMenuFlyoutItemViewModel> GetLocationItemMenuItems(INavigatio
...
-                    ShowItem = showMoveItemDown
+                    ShowItem = showMoveItemDown 
                 },
                 new ContextMenuFlyoutItemViewModel()
                 {
                     Text = "SideBarFavoritesMoveToBottom".GetLocalized(),
                     Glyph = "\uE118",
                     Command = MoveItemToBottomCommand,
-                    ShowItem = showMoveItemDown
+                    ShowItem = showMoveItemDown
```
Comment (yaichenbaum) [inline src/Files.Uwp/UserControls/SidebarControl.xaml.cs:278]: Can you remove the extra space?

File: src/Files.Uwp/ViewModels/Previews/BasePreviewModel.cs [hunk 10]
```diff
@@ -131,22 +131,16 @@ private async Task<List<FileProperty>> GetSystemFilePropertiesAsync()
...
-            if (userSettingsService.PreferencesSettingsService.AreFileTagsEnabled)
-            {
-                list.FirstOrDefault(x => x.ID is "filetag").Value = 
-                    Item.FileTagsUI is not null ? string.Join(',', Item.FileTagsUI.Select(x => x.TagName)) : null;
-            }
-            else
-            {
-                _ = list.Remove(list.FirstOrDefault(x => x.ID is "filetag"));
-            }
+
```
Comment (yaichenbaum) [inline src/Files.Uwp/ViewModels/Previews/BasePreviewModel.cs:134]:
```suggestion
```

File: src/Files.Uwp/Helpers/ContextFlyoutItemHelper.cs (hunk 6 above)
Comment (ferrariofilippo) [reply src/Files.Uwp/Helpers/ContextFlyoutItemHelper.cs:294]: Yes, I'll fix it

File: src/Files.Uwp/Filesystem/ListedItem.cs (hunk 5 above)
Comment (ferrariofilippo) [reply src/Files.Uwp/Filesystem/ListedItem.cs:136]: Just checked, the property still return 'null' if there are no tags selected.
Functions that use this property check if the value is null

File: src/Files.Uwp/Helpers/ContextFlyoutItemHelper.cs [hunk 11]
```diff
@@ -441,7 +440,7 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseLayoutMenuItems(Curren
                         {
                             Text = "FileTags".GetLocalized(),
                             IsChecked = currentInstanceViewModel.FolderSettings.DirectoryGroupOption == GroupOption.FileTag,
-                            ShowItem = userSettingsService.PreferencesSettingsService.AreFileTagsEnabled,
+                            ShowItem = true,
```
Comment (yaichenbaum) [inline src/Files.Uwp/Helpers/ContextFlyoutItemHelper.cs:443]: Same as above

File: src/Files.Uwp/Strings/en-US/Resources.resw [hunk 12]
```diff
@@ -2345,12 +2345,6 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="DetailsViewHeaderFlyout_ShowFileTag.Text" xml:space="preserve">
     <value>Tag</value>
   </data>
-  <data name="SettingsEditFileTagsExpander.Title" xml:space="preserve">
-    <value>Edit tags</value>
```
Comment (yaichenbaum) [inline src/Files.Uwp/Strings/en-US/Resources.resw:2349]: I think we still need this string, can you restore it?

File: src/Files.Uwp/UserControls/SidebarControl.xaml.cs [hunk 13]
```diff
@@ -261,14 +261,14 @@ private List<ContextMenuFlyoutItemViewModel> GetLocationItemMenuItems(INavigatio
...
-                    ShowItem = showMoveItemUp
+                    ShowItem = showMoveItemUp 
                 },
                 new ContextMenuFlyoutItemViewModel()
                 {
                     Text = "SideBarFavoritesMoveOneDown".GetLocalized(),
                     Glyph = "\uE70D",
                     Command = MoveItemDownCommand,
-                    ShowItem = showMoveItemDown
+                    ShowItem = showMoveItemDown
```
Comment (yaichenbaum) [inline src/Files.Uwp/UserControls/SidebarControl.xaml.cs:271]:
```suggestion
                    ShowItem = showMoveItemDown
```

File: src/Files.Uwp/UserControls/SidebarControl.xaml.cs [hunk 14]
```diff
@@ -261,14 +261,14 @@ private List<ContextMenuFlyoutItemViewModel> GetLocationItemMenuItems(INavigatio
                     Text = "SideBarFavoritesMoveOneUp".GetLocalized(),
                     Glyph = "\uE70E",
                     Command = MoveItemUpCommand,
-                    ShowItem = showMoveItemUp
+                    ShowItem = showMoveItemUp
```
Comment (yaichenbaum) [inline src/Files.Uwp/UserControls/SidebarControl.xaml.cs:264]:
```suggestion
                    ShowItem = showMoveItemUp
```

**Comment by yaichenbaum** [comment]
@ferrariofilippo last thing, can you add the link icon back to the button? It should look like this.
image

**Comment by ferrariofilippo** [comment]
Ok, I fixed it

File: src/Files.Uwp/Views/SettingsPages/Experimental.xaml [hunk 15]
```diff
@@ -34,33 +34,25 @@
...
-                            <FontIcon
-                                FontSize="14"
-                                Foreground="{ThemeResource TextFillColorPrimaryBrush}"
-                                Glyph="&#xE104;" />
-                        </local:SettingsBlockControl.Icon>
+            
+            <!--  Edit File Tags  -->
+            <local:SettingsBlockControl Title="{helpers:ResourceString Name=SettingsEditFileTagsExpander/Title}" HorizontalAlignment="Stretch"
+                ButtonCommand="{Binding EditFileTagsCommand}"
+                IsClickable="True">
```
Comment (yaichenbaum) [inline src/Files.Uwp/Views/SettingsPages/Experimental.xaml:55]:
```suggestion


```

Comment (yaichenbaum) [inline src/Files.Uwp/Views/SettingsPages/Experimental.xaml:42]:
```suggestion
```

File: src/Files.Uwp/Views/SettingsPages/Experimental.xaml [hunk 16]
```diff
@@ -34,33 +34,24 @@
                 FontStyle="Italic"
                 Text="{helpers:ResourceString Name=SettingsExperimentalDescription/Text}" />
 
-            <local:SettingsBlockControl
-                Title="{helpers:ResourceString Name=SettingsEnableFileTags/Title}"
-                HorizontalAlignment="Stretch">
+
```
Comment (yaichenbaum) [inline src/Files.Uwp/Views/SettingsPages/Experimental.xaml:37]:
```suggestion
```

File: src/Files.Uwp/Views/SettingsPages/Experimental.xaml [hunk 17]
```diff
@@ -34,33 +34,23 @@
...
-                </local:SettingsBlockControl.ExpandableContent>
+                            Glyph="&#xE104;" />
+                </local:SettingsBlockControl.Icon>
+                <FontIcon
+                    FontSize="14"
+                    Foreground="{ThemeResource TextFillColorPrimaryBrush}"
+                    Glyph="&#xE2B4;" />
             </local:SettingsBlockControl>
-
+
```
Comment (yaichenbaum) [inline src/Files.Uwp/Views/SettingsPages/Experimental.xaml:52]:
```suggestion
```

File: src/Files.Uwp/Views/SettingsPages/Experimental.xaml [hunk 18]
```diff
@@ -34,33 +34,22 @@
...
-
-                        <FontIcon
+                    <FontIcon
                             FontSize="14"
                             Foreground="{ThemeResource TextFillColorPrimaryBrush}"
-                            Glyph="&#xE2B4;" />
-                    </local:SettingsBlockControl>
-                </local:SettingsBlockControl.ExpandableContent>
+                            Glyph="&#xE104;" />
+                </local:SettingsBlockControl.Icon>
```
Comment (yaichenbaum) [inline src/Files.Uwp/Views/SettingsPages/Experimental.xaml:46]:
```suggestion
                </local:SettingsBlockControl.Icon>

```

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Comment by yaichenbaum** [comment]
@ferrariofilippo excellent work! Thank you for contributing to Files, this change will be included in the next release.

File: src/Files.Uwp/Helpers/ContextFlyoutItemHelper.cs (hunk 11 above)
Comment (Phanhasomlout) [reply src/Files.Uwp/Helpers/ContextFlyoutItemHelper.cs]: - ### **[](url)**

**Pull request merged**